cmake/AutoOptionHelpers.cmake

```diff
@@ -21,17 +21,20 @@ set(GLOBAL_AUTO_OPTION_VARS "")
 
 function(_REGISTER_AUTO_OPTION _NAME _FEATURE_VAR _DESCRIPTION _DEFAULT)
   add_custom_target(${_NAME}_target)
-  set_target_properties(${_NAME}_target
-    PROPERTIES
-      AUTO_OPTION_FEATURE_VAR ${_FEATURE_VAR}
-  )
+  set_target_properties(${_NAME}_target PROPERTIES AUTO_OPTION_FEATURE_VAR ${_FEATURE_VAR})
```

Comment:
What's with the formatting? This was changed to be on one line, while line 29 was changed to be split.

Reply:
The cmake formatter is a fickle beast

Reply:
I thought about this a little more. If the formatter is not run from the project root, it presumably will not detect our config and will use its defaults. We need a way to be confident this is actually our config and not the default one. I think we ran the formatter across the entire repository when we first started using it, so this module should **not** need to be reformatted now.

Reply:
Having a pre-commit hook and a CMake target like we do for clang-format would help with this.